docs/guide.md

```diff
@@ -535,7 +537,7 @@ Now we can add the Firebase deployment configuration to our `package.json` like
 }
 ```
 
-Change the `<user>` part in `"name": "donejs-chat-<user>"` to your GitHub username. This will be your Firebase application name (you can choose any other available name as well).
+Change the `<appname>` to the name of the application created when you set up the Firebase app.
 
 And also update the production `baseURL` in the `system` section:
```

Comment:
You have to add envs, server-production, and baseURL right?  Wording makes it seem like you just have to add baseURL.